### PR TITLE
[REV-1045] Decrease perception of loading time on left side

### PR DESCRIPTION
**Pull request by MatthewPiatetsky**
before:
loading2
after:
loading3

### CODE REVIEW

**Comment by emma-green** [comment]
Perhaps I missed something, but what's the point of not showing the skeleton? Wouldn't that increase the perception of loading time?

**Comment by MatthewPiatetsky** [comment]
@emma-green well right now the process is
1. page loads with skeleton on left side
2. wait for full left side to load
3. show page

I changed it to
1. Page loads with in your cart
2. Product details appears when it is ready
3. Order Details appears when it is ready
4. Order summary appears when it is ready

The skeleton is rather complex, so adding a skeleton for each of those 3 sections would take several days. For juliana's suggestion, we could potentially only add a single spinner on the order summary, that would certainly be simpler. We could certainly add the skeleton for each section, would just take a while.
Could you clarify your point though about how this would increase perception of loading time?
Things popping in when they're ready appears to load  quicker i think?

File: src/payment/cart/Cart.jsx [hunk 1]
```diff
@@ -78,9 +89,11 @@ class Cart extends React.Component {
             <CouponForm isPaymentVisualExperiment={isPaymentVisualExperiment} /> : null}
           <TotalTable total={orderTotal} />
           {isCurrencyConverted ? <CurrencyDisclaimer /> : null}
-        </OrderSummary>
+        </OrderSummary> : (<div><div className="skeleton py-2 mb-3 w-50" />
```
Comment (julianajlk) [inline src/payment/cart/Cart.jsx]: You can also add <React.Fragment></React.Fragment> if you don't want to add the extra < div> here, so you can still group multiple children without adding the extra node.

File: src/payment/cart/order-details/OrderDetails.jsx [hunk 2]
```diff
@@ -85,7 +85,10 @@ class OrderDetails extends Component {
 
   render() {
     if (this.props.messageType === null) {
-      return null;
+      return (this.props.REV1045Experiment ? <div><div className="skeleton py-2 mb-3 w-50" />
```
Comment (julianajlk) [inline src/payment/cart/order-details/OrderDetails.jsx]: Same as the above, could replace < div> with <React.Fragment>

**Pull request merged**